src/main/java/frc/robot/subsystems/ElevatorSubsystem.java

```diff
@@ -0,0 +1,144 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.configs.CurrentLimitsConfigs;
+import com.ctre.phoenix6.configs.MotorOutputConfigs;
+import com.ctre.phoenix6.configs.SoftwareLimitSwitchConfigs;
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.math.controller.ElevatorFeedforward;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.units.measure.MutAngle;
+import edu.wpi.first.units.measure.MutAngularVelocity;
+import edu.wpi.first.units.measure.MutVoltage;
+import edu.wpi.first.units.measure.Voltage;
+import edu.wpi.first.wpilibj.RobotController;
+import edu.wpi.first.wpilibj.sysid.SysIdRoutineLog;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+
+public class ElevatorSubsystem extends SubsystemBase {
+  // This gearbox represents a gearbox containing 4 Vex 775pro motors.
+  private final double ELEVATOR_KP = 0.1;
+  private final double ELEVATOR_KI = 0;
+  private final double ELEVATOR_KD = 0;
+  private final double ELEVATOR_KS = 0;
+  private final double ELEVATOR_KV = 0;
+  private final double ELEVATOR_KG = 0;
+  private final double ELEVATOR_KA = 0;
+  private final double reverseSoftLimit = -67;
+  private final double forwardSoftLimit = -1;
+
+  // Standard classes for controlling our elevator
+  private final ProfiledPIDController m_controller =
```

Comment:
Since this is a CTRE-based motor controller (the TalonFX is made by CTRE), using https://v6.docs.ctr-electronics.com/en/latest/docs/api-reference/device-specific/talonfx/motion-magic.html#motion-magic-expo should get better results since the PID & motion profile run on the motor itself at a much higher rate than the Rio can run code & update the motor.